zuul.d/project-templates.yaml

```diff
@@ -86,7 +86,7 @@
               ansible_test_collections: true
         - ansible-test-network-integration-asa-python35:
             vars:
-              ansible_test_collections: true
+              ansible_test_collections: false
```

Comment:
Don't change this, add voting: false like:

  https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L180